JS/Lesson22/sinan/style.css

```diff
@@ -0,0 +1,63 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins:200,300,400,500,600,700,800,900&display=swap');
```

Comment:
Consider importing fonts in HTML instead of CSS. It will help you follow best practices and optimize the loading of your web page. Here are some reasons why you should consider importing fonts in HTML rather than CSS:

- Faster Rendering: Placing the font import directly in the HTML using the <link> or <style> tag in the <head> of your HTML document ensures that the browser loads the font early in the rendering process. This can lead to faster font rendering, reducing the chance of the "Flash of Unstyled Text" (FOUT) or "Flash of Invisible Text" (FOIT) that can occur when fonts are loaded asynchronously via CSS.

- Improved User Experience: By loading fonts early in the rendering process, you provide a better user experience. Users will see text rendered in the desired font sooner, even if other CSS styles are still being loaded.

- Preventing FOIT: Loading fonts in HTML can help prevent FOIT, where text remains invisible until the custom font is fully downloaded. This is especially important for critical text like headings or important content.

- Better Browser Compatibility: Some older browsers may not fully support @font-face rules in CSS or may have issues with how they're applied. Importing fonts in HTML is a more straightforward and reliable method that works across a wide range of browsers.